tests/test_other.py

```diff
@@ -4074,9 +4031,11 @@ def test_returncode(self):
       for no_exit in [0, 1]:
         for call_exit in [0, 1]:
           for async in [0, 1]:
-            subprocess.check_call([PYTHON, EMCC, 'src.cpp', '-DCODE=%d' % code, '-s', 'NO_EXIT_RUNTIME=%d' % no_exit, '-DCALL_EXIT=%d' % call_exit, '-s', 'BINARYEN_ASYNC_COMPILATION=%d' % async])
+            run_process([PYTHON, EMCC, 'src.cpp', '-DCODE=%d' % code, '-s', 'NO_EXIT_RUNTIME=%d' % no_exit, '-DCALL_EXIT=%d' % call_exit, '-s', 'BINARYEN_ASYNC_COMPILATION=%d' % async])
             for engine in JS_ENGINES:
-              if async and engine == V8_ENGINE: continue # async compilation can't return a code in d8
+              # async compilation can't return a code in d8
```

Comment:
btw, do you happen to know if this is still true?

Reply:
No, but I can take a look.  We should probably open a bug and include a link here if it is.